lang/po/Firmware_ro.po

```diff
@@ -584,16 +584,16 @@ msgid ""
 "Ensure filament can move and FINDA works."
 msgstr ""
 "FINDA nu sa oprit la descarcare Fil.Incercati descar. manuala. "
-"Asig. ca FINDA functioneaza."
+"Asigurativa ca FINDA functioneaza."
 
 #. MSG_DESC_FINDA_DIDNT_TRIGGER c=20 r=8
 #: ../../Firmware/mmu2/errors_list.h:196 ../../Firmware/mmu2/errors_list.h:236
 msgid ""
 "FINDA didn't trigger while loading the filament. Ensure the filament can "
 "move and FINDA works."
 msgstr "FINDA nu sa declansat in timpul incarcarii"
-"Asigurativa ca filam. se misca si FINDA functioneaza"
-"
+"Asigurativa ca filamentul se misca si FINDA functioneaza"
```

Comment:
Again 
msgstr ""
and then the translation

Reply:
@3d-gussner 
like this below I see now I have for a few dont know how I change it 
msgstr ""
"FINDA nu sa declansat in timpul incarcarii"
"Asigurativa ca filamentul se misca si FINDA functioneaza"